core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -117,6 +117,16 @@ case class CreateRawlsBillingProjectFullRequest(
   privateIpGoogleAccess: Option[Boolean],
   servicePerimeter: Option[ServicePerimeterName])
 
+// V2 billing projects will have enable flow logs on if in a service perimeter
```

Comment:
TOL: We will need to re-evaluate this if service perimeters are ever used more widely. Flow logs and service perimeters are only really tied together because of how AoU manages their perimeter. A non-AoU perimeter would not necessarily require flow logs to be on.